js/repl/public/index.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
```

Comment:
@bvaughn  - Should be pretty straightforward. You just need to make a HTML file with "YAML front matter" so that Jekyll processes it. See how the old REPL does it: https://github.com/babel/babel.github.io/blob/master/repl.html